### ISSUE TITLE
[Bug] Discrepancy in point label format for iOS/Android

### ISSUE THREAD

**Issue by PeterStaev** [telerik/nativescript-ui-feedback#11]
I have a Cartesian chart that shows point labels. The data bound to the chart contains whole numbers. On iOS using either `textFormat="%i"` or `textFormat="%d"` shows correctly the numbers. On Android those formats crash the app. For android `textFormat="%.0f"` works. But the same on iOS shows some huge numbers.

**Comment by NickIliev** [comment]
+1 reported via **t.1117182**

Issue: textFormat is not working as meant by [the specification.](http://pubs.opengroup.org/onlinepubs/009695399/functions/printf.html).

For example in iOS `textFormat="%.2f"` does not show the right value but the same owrks fine for Android.

**Comment by chuckmitchell** [comment]
Just encountered this one on iOS.

**Comment by tgpetrov** [comment]
@chuckmitchell Indeed, currently there is a difference in the native implementation of the chart. 

On Android all numbers are treated like decimal numbers (which work well with the "%.2f" format, or "%.0f" to format them as whole numbers) while on iOS the numbers preserve the type of the data you pass. That means that on iOS you can use the "%d" or "%i" formats for whole numbers, and the "%.2f"/"%.0f" for decimal numbers. 

To get a consistent behavior across platforms you can ensure that the number you pass to the chart are all decimal numbers (instead of "5", use "5.0") and use the floating formats like ("%.2f" or "%.0f") on both platforms.
